### PR TITLE
major: 0.3 release preparation

### PR DESCRIPTION
**Pull request by Vovke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,132 @@
-## A gateway daemon for Kalatori
+## A Gateway Daemon for Kalatori
 
+Kalatori is an open-source daemon designed to enable secure and scalable blockchain payment processing. Licensed under GPLv2 ([LICENSE](LICENSE)), Kalatori currently supports assets on the Polkadot relay chain and its parachains.
```
Comment (Slesarew) [inline README.md:3]:
```suggestion
Kalatori is an open-source daemon designed to enable secure and scalable blockchain payment processing. Licensed under GPLv3 ([LICENSE](LICENSE)), Kalatori currently supports assets on the Polkadot relay chain and its parachains.
```

**Comment by Vovke** [comment]
follow up issue #151 to update outdated docs

File: README.md (hunk 1 above)
Comment (Vovke) [reply README.md]: resolved in d6334fd

File: README.md [hunk 2]
```diff
@@ -1,101 +1,132 @@
...
-Use the following command inside this repository root directory to run Chopstick with the Polkadot config example:
+We welcome contributions! Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for guidelines on contributing and submitting pull requests.
 
-```sh
-npx @acala-network/chopsticks@latest -c chopsticks/pd.yml
-```
+### License
 
-Then run `kalatori` with `KALATORI_RPC` set on the Chopsticks default server:
+Kalatori is open-source software licensed under the GPLv2 License. See the [LICENSE](LICENSE) file for more details.
```
[the remark this answers is not in the document]
Comment (Vovke) [reply README.md]: resolved in d6334fd

File: README.md [hunk 3]
```diff
@@ -1,101 +1,132 @@
-## A gateway daemon for Kalatori
+## A Gateway Daemon for Kalatori
 
+Kalatori is an open-source daemon designed to enable secure and scalable blockchain payment processing. Licensed under GPLv2 ([LICENSE](LICENSE)), Kalatori currently supports assets on the Polkadot relay chain and its parachains.
+
+The daemon derives unique accounts for each payment using a provided seed phrase and outputs all payments to a specified recipient wallet. It also offers limited transaction tracking for order management. Kalatori operates in a single-threaded mode and supports multiple currencies configured in a simple TOML-based configuration file.
```
[the remark this answers is not in the document]
Comment (Vovke) [reply README.md]: resolved in d6334fd

**Review by Slesarew — APPROVED** [review]

**Review by Slesarew — APPROVED** [review]

**Pull request merged**